### PR TITLE
New Version: Sigil-Ebook.PageEdit version 2.3.1

### PR DESCRIPTION
**Pull request by dougmassay**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/172238)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Programs\PageEdit\PageEdit.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Programs\PageEdit\QtWebEngineProcess.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 895.)

**Comment by dougmassay** [comment]
See the previous version's PR: #168593

There's a weird catch-22 where the local scope install can't find the Microsoft.VCRedist.2015+.x64 dependency, but there is no Microsoft.VCRedist.2015+.x64 package available for the local scope. This same scenario was approved last time. It tests fine locally since Microsoft.VCRedist.2015+.x64 is installed by default on the Minimum OS version this manifest supports. Thanks for your time and efforts!

File: manifests/s/Sigil-Ebook/PageEdit/2.3.1/Sigil-Ebook.PageEdit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  InstallerSha256: 332B562614B2EF81D7E6D3B6F73D4CC4D1D1E304015CDE9F3D52F705344AE5AC
+  InstallerSwitches:
+    Custom: /ALLUSERS /NORESTART /SUPPRESSMSGBOXES /COMPONENTS="dicon,*afiles"
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/Sigil-Ebook/PageEdit/releases/download/2.3.1/PageEdit-2.3.1-Windows-x64-Setup.exe
+  InstallerSha256: 332B562614B2EF81D7E6D3B6F73D4CC4D1D1E304015CDE9F3D52F705344AE5AC
```
Comment (stephengillie) [inline manifests/s/Sigil-Ebook/PageEdit/2.3.1/Sigil-Ebook.PageEdit.installer.yaml:32]:
```suggestion
  InstallerSha256: 332B562614B2EF81D7E6D3B6F73D4CC4D1D1E304015CDE9F3D52F705344AE5AC
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @dougmassay,

It seems that both scopes need the dependency:
image

**Comment by dougmassay** [comment]
We've been down this road again and again. Yes, both scopes need the dependency, but the validation process fails miserably if the dependency is specified for the user scope installation the manifest (because no user scope package exists for Microsoft.VCRedist.2015+.x64). 

Hence the catch-22 I mentioned.

**Comment by dougmassay** [comment]
I will, however, try your suggested change.

**Comment by dougmassay** [comment]
Someone needs to explain how we're supposed properly manifest a user scope install that has a Microsoft.VCRedist.2015+.x64 dependency, because I'm tired of guessing.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978334972
> 2024-09-10 06:39:54.191 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.40.33810.0

Automated error analysis suggests -1978334972 may mean This package has a dependency missing from your system.

(Automated response - build 895.)

**Review by stephengillie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
[Policy] Needs-Manual-Merge

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**